src/cljs/rems/administration/resource.cljs

```diff
@@ -148,18 +156,34 @@
    (text :t.administration/cancel)])
 
 (defn create-resource-page []
```

Comment:
This function does quite a lot of things, could there be a name that would describe it better? Or splitting it to smaller functions?

Reply:
Hmm, to me it looks like a pretty simple container component that subcribes to some state and renders itself using a few other components. I don't see how this can be simplified. Do you have some ideas?

Reply:
Maybe it is then just my unfamiliariness of cljs ui code.